src/components/hero.scss

```diff
@@ -19,8 +19,9 @@
       font-style: normal;
       font-weight: 800;
       font-size: 64px;
-      line-height: 70px;
+      line-height: 1.1;
       flex-wrap: wrap;
+      max-width: 517px;
```

Comment:
question: What does 517px mean? It's good to leave a code comment explaining the way of this "magic" numbers so that other developers also understand your code :)

Reply:
First, It is the size mentionned pn the design. Then, it allows to break the sentence avoiding using <br>